Documentation/Blazorise.Docs/Pages/Blog/2024-05-17_HowToCreateSocialMediaShareButtons/Index.md

```diff
@@ -0,0 +1,205 @@
+﻿---
+title: How to create social media share buttons
+description: Discover how to create share buttons for your Blazor app!
+permalink: /blog/how-to-create-social-media-share-buttons
+canonical: /blog/how-to-create-social-media-share-buttons
+image-url: /img/blog/2024-05-17/how-to-create-social-media-share-buttons.png
+image-title: How to create social media share buttons with Blazorise
+author-name: Giorgi
+author-image: giorgi
+posted-on: May 21st, 2024
+read-time: 5 min
+---
+
+# How to create a ShareButton component with Blazorise!
+
+Are you ready to sprinkle some Blazorise magic into your Blazor app? Adding share buttons for social media platforms can give your users an easy way to spread the word about your awesome content. It's easier than you think, thanks to Blazorise!
+
+---
+
+## Let's dive in and jazz up your app with these fantastic buttons.
+
+## Installing Blazorise
```

Comment:
I'm not sure if we need an installation guide here. It is better to forward users to another blog post explaining it(https://localhost:5001/blog/how-to-create-a-blazorise-application-beginners-guide), or link a documentation pages.